.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,31 @@
+name: "\U0001F680 Feature request:"
```

'Feature request:' the only heading with a colon proceeding it
```suggestion
name: "\U0001F680 Feature request"
```